tee-worker/litentry/core/vc-task/receiver/src/mock.rs

```diff
@@ -0,0 +1,229 @@
+use super::*;
+use base58::FromBase58;
+use codec::Decode;
+use core::time::Duration;
+use ita_stf::AesOutput;
+use itp_extrinsics_factory::mock::ExtrinsicsFactoryMock;
+use itp_node_api::metadata::{metadata_mocks::NodeMetadataMock, provider::NodeMetadataRepository};
+use itp_sgx_crypto::{mocks::KeyRepositoryMock, ShieldingCryptoDecrypt, ToPubkey};
+use itp_stf_executor::mocks::StfEnclaveSignerMock;
+use itp_stf_primitives::{traits::TrustedCallSigning, types::KeyPair};
+use itp_test::mock::{
+	handle_state_mock::HandleStateMock, onchain_mock::OnchainMock,
+	shielding_crypto_mock::ShieldingCryptoMock,
+};
+use itp_top_pool_author::{
+	error::Result,
+	mocks::{AuthorApiMock, GLOBAL_MOCK_AUTHOR_API, GLOBAL_MOCK_RPC_API},
+};
+use itp_types::MrEnclave;
+use lazy_static::lazy_static;
+use lc_data_providers::DataProviderConfig;
+use lc_stf_task_sender::{AssertionBuildRequest, RequestType};
+use litentry_primitives::{aes_decrypt, AesRequest, Assertion, RequestAesKey};
+use sp_core::{blake2_256, sr25519, Pair};
+use std::{sync::mpsc::Receiver, vec::Vec};
+
+pub const COMMON_SEED: &[u8] =
+	b"crouch whisper apple ladder skull blouse ridge oven despair cloth pony";
+lazy_static! {
+	pub static ref SHIELDING_KEY: ShieldingCryptoMock = ShieldingCryptoMock::default();
+}
+
+pub fn init_global_mock_author_api() -> Result<Receiver<Vec<u8>>> {
```

Comment:
What do you think about keeping only mock definitions here and removing test util functions and functions that initializes tests context (move everything to `test.rs` and remove this file) ?

Reply:
Yeah I can do that, I followed the pattern of unit tests in pallets.